scopes/dns/setup_dns.tf

```diff
@@ -0,0 +1,15 @@
+data "google_dns_managed_zone" "lpt" {
+  name = "lpt"
+}
+
+resource "google_dns_record_set" "creditapplication-service" {
+  name         = "creditapplication.api.lpt.senacor.com."  # Replace with your subdomain
+  managed_zone = data.google_dns_managed_zone.lpt.name
+  type         = "A"
+  ttl          = 300
+  rrdatas      = [google_compute_global_address.creditapplication_service_ingress_ip.address]
+}
+
+resource "google_compute_global_address" "creditapplication_service_ingress_ip" {
```

Comment:
ist das dann nicht data wenn schon da?

Reply:
Ich habe nachgeforscht und die Reihenfolge ist doch eine Andere. Zuerst muss diese eine statische IP in GCP hinterlegt werden, und dann wird beim k8s-Deployment, dank der Annotation aus deinem PR, der Ingress mit der IP verknüpft. D.h. wir brauchen den separaten Scope nicht, also ich zieh's um nach `scopes/application`